en/developer_guide/routing.rst

```diff
@@ -102,27 +114,32 @@ Declaring differents loaders as services in ``routing.yml`` is mandatory to gene
         resource: '.'
         type: orchestra_redirection
 
-See `Symfony Loader documentation`_ for more information.
+You also have to modify the project configuration, in the ``config.yml`` file :
+
+.. code-block:: yaml
```

Comment:
I think that it could be visible once merged, I don't see any differences with other pages...